File: growth_tracker/lib/models/reminder.dart
class Reminder {

  final int id;
  final String title;
  final String description;
  final DateTime reminderDate;
  bool isCompleted;

  Reminder({
    required this.id,
    required this.title,
    required this.description,
    required this.reminderDate,
    this.isCompleted = false,
  });

  factory Reminder.fromJson(Map<String, dynamic> json) {
    return Reminder(
      id: json['id'],
      title: json['title'],
      description: json['description'],
      reminderDate: DateTime.parse(json['reminderDate']).toLocal(),
      isCompleted: json['isCompleted'],
    );
  }
}

File: GrowthTracker/Models/Reminder.cs
using System;

namespace GrowthTracker.API.Models;

public class Reminder
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTime ReminderDate { get; set; }
    public bool IsCompleted { get; set; }
}
